core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -166,8 +166,10 @@ public void useSameContextWithZookeeper() throws Exception {
    */
   @Test(expected = IllegalArgumentException.class)
   public void cannotInitializeWithoutMasterHostname() throws Exception {
-    URI uri = URI.create(Constants.HEADER + "/tmp/path.txt");
-    org.apache.hadoop.fs.FileSystem.get(uri, getConf());
+    try (Closeable c = new ConfigurationRule(PropertyKey.MASTER_HOSTNAME, null).toResource()) {
```

Comment:
Can this not be done with an annotation?

Reply:
Do you mean no using `@Test(expected=Exception.class)` ?

Reply:
Oh I meant `@Config`, but I think that only works for integration tests, so this is fine.